Source/UnrealExamples/Public/WebMessage.h

```diff
@@ -0,0 +1,21 @@
+// Fill out your copyright notice in the Description page of Project Settings.
+
+#pragma once
+
+#include "CoreMinimal.h"
+
+class UNREALEXAMPLES_API WebMessage
+{
+public:
+	FString Type;
```

Comment:
is this field used?

Reply:
it's not currently used but if a dev wants to check if it's a guest or logged in user they could use this